testing/test_xspec.py

```diff
@@ -137,6 +138,7 @@ def getnice(channel):
         gw.exit()
         if remotenice is not None:
             gw = makegateway("popen//nice=5")
+            time.sleep(0.1)
             remotenice2 = gw.remote_exec(getnice).receive()
             assert remotenice2 == remotenice + 5
```

Comment:
do you have a supporting traceback for changing this line?

Reply:
It is part of the first commit's message.

Reply:
I uploaded a full log at https://www.zq1.de/~bernhard/temp/python-execnet-build.log
To trigger it, I generated some parallel load with `perl -e 'fork;fork;fork;while(1){}'`